db/db.go

```diff
@@ -79,7 +81,7 @@ func Init(config config.DBConfig, debug bool) *Database {
 		db.Config.Logger.LogMode(logger.Info)
 	}
 
-	d := &Database{db}
+	d := &Database{DB: db}
```

Comment:
Not sure this change is necessary (though it doesn't hurt)

Reply:
I had _InvalidStructLit_ error from gopls without it since I added `lock sync.Mutex` field